plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, read the values of `nextChildBlock` and `childBlockInterval` from the contract (e.g. via [`eth_call`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call)). Compute block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read values of roots of Merkle trees of blocks by calling `PlasmaFramework.blocks(blknum)`.
```

Comment:
`(e.g. via [`eth_call`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call)`

hm, not the thing that is needed here to answer this:
> how to get childBlockInterval and nextChildBlock

Not asking for general ethereum-integration advice, but on the level of the contract code and defining the values you're using. What API one needs to hit, which contract? If applicable, something along the lines of `PlasmaFramework.blocks(blknum)` you did.

Please be mindful about that your `nextChildBlock` and `childBlockInterval` only appear in your added section. A reader, who doesn't know what those are, will not know how to "integrate".